gguf-py/gguf/gguf_writer.py

```diff
@@ -299,6 +300,7 @@ def write_tensors_to_file(self, *, progress: bool = False) -> None:
                     tensor.tofile(self.fout)
                     bar.update(tensor.nbytes)
                     self.write_padding(self.fout, tensor.nbytes)
+                    del tensor
```

Comment:
Looking back on this, yeah, that makes no sense. Considering `tensor` is `pop`ped from the list, there shouldn't be a reference left to it if everything else works as intended, and there isn't - commenting out this line shows no change in memory usage.

I think I was conflating cause and effect of a few different things when I made this fix. But since there's no harm in leaving it, it'll stay. Thanks for pointing that out.